src/data-table/DataTableBody.tsx

```diff
@@ -94,7 +94,7 @@ export function DataTableBody<T extends ReferenceObject>(props: DataTableBodyPro
     function createRow(row: T, index: number | string, level = 0) {
         const labelId = `data-table-row-${index}`;
         const defaultAction = parseActions([row], availableActions)[0];
```

Comment:
Not sure if we actually need ``defaultAction`` for anything right now. Its only usage is to show/hide the three button icon.

```suggestion
        const hasRowActions = parseActions([row], availableActions).length > 0;
```